app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1286,9 +1288,27 @@ public void onBackPressed() {
             getSupportFragmentManager().popBackStack(NARROW,
                     FragmentManager.POP_BACK_STACK_INCLUSIVE);
         } else {
-            super.onBackPressed();
+            if(atHome){
```

Comment:
@kunall17  Sure. This can also help to keep a check. 👍  Will be implementing it.